### ISSUE TITLE
Consider rewriting 'references to external resources' processing mode feature

### ISSUE THREAD

**Issue by nikosandronikos** [w3c/svgwg#245]
For the '**references to external resources**' section of [Processing modes](https://svgwg.org/svg2-draft/conform.html#processing-modes) / [Features](https://svgwg.org/svg2-draft/conform.html#features), we should consider rewriting this section to refer to Fetch as we have with our other resource handling rewrites.

The issue was originally inline in the spec as:
> This is all too handwavy. And we perhaps shouldn't try to make an exhaustive list. This needs to be defined in terms of Fetch, probably. And the URL Standard for comparing the URLs.

**Comment by AmeliaBR** [comment]
Yeah, we can now probably integrate it better with the main section on URL handling that I expanded on in the linking chapter (https://svgwg.org/svg2-draft/linking.html#processingURL).
